control-plane/pkg/reconciler/trigger/trigger_lifecycle.go

```diff
@@ -29,6 +29,10 @@ import (
 	"knative.dev/eventing-kafka-broker/control-plane/pkg/contract"
 )
 
+const (
+	ConditionConsumerGroup apis.ConditionType = "ConsumerGroupReady"
+)
+
```

Comment:
This seems to be unused, did you intend to use it?

Reply:
Was using it earlier. I'll keep it out for now till we may need a new condition. Thanks